newIDE/app/src/UI/Theme/themes.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Can you explain where this file is used? Is it somehow doing a duplicate job with ThemeRegistry.js?

Reply:
Yeah sure! It does look pretty similar.   
This file is only used by the npm scripts to generate the rest of the files. `build-theme-styles` uses it to locate all of the `theme.json`s, and `create-new-theme` reads it in order to re-write the ThemeRegistry, which is the file that's actually used by the application.

Reply:
It could for sure have a better name, and I'm thinking it might do better to live with those scripts instead?

Reply:
If it's not used at all in the source of the app, then yep it's better to have it next to the scripts. Though I wonder: would it be possible to do without it?
`build-theme-styles` could get a list of folders (inside `newIDE/app/src/UI/Theme`) and read the `theme.json` inside each folder? This way there is no way to maintain a separate file: the folder ARE the source of truth, which is simpler and more robust :)

Reply:
Alright, `themes.json` is gone now. Now it's all dependent on the ThemeRegistry, so there's no duplication ✅